src/core/sendValidator.js

```diff
@@ -201,8 +201,8 @@ class SendValidator {
     
     /**
      * Check if the message has a Message Body
-     * If an Api Template is specified it will override the HtmlBody and/or the textBody.
-     * If no Api Template is specified the HtmlBody and/or the textBody must me set
+     * If an Api Template is specified it will override the HtmlBody, AmpBody, and/or the textBody.
+     * If no Api Template is specified the HtmlBody, AmpBody, and/or the textBody must me set
```

Comment:
rephrase this line: "If no Api Template is specified the HtmlBody and/or the PlainTextBody must be set"